cli/src/cmd/install.rs

```diff
@@ -79,12 +79,19 @@ pub(crate) fn install(
 
 /// installs the dependency as an ordinary folder instead of a submodule
 fn install_as_folder(dep: &Dependency, libs: &Path) -> eyre::Result<()> {
-    Command::new("git")
+    let result = Command::new("git")
         .args(&["clone", &dep.url, &dep.name])
         .current_dir(&libs)
         .stdout(Stdio::piped())
         .spawn()?
-        .wait()?;
+        .wait();
+
+    // exiting if the repo doesn't exist
+    if let Ok(status) = result {
+        if status.code() == Some(128) {
+            eyre::bail!("Repo: \"{}\" not found!", &dep.url);
+        }
+    }
```

Comment:
With the above change this is just `if status.code() == ...`.

An alternative is using `.status()?.exit_ok().map_err(..)?` which would also catch exit code 1 (which is used in some cases by git)

Reply:
Was wondering about other error codes, couldn't find any docs online, making changes!